src/Base/QuantityLexer.c

```diff
@@ -56,7 +56,7 @@
 #if defined (__STDC_VERSION__) && __STDC_VERSION__ >= 199901L
 
 /* C99 says to define __STDC_LIMIT_MACROS before including stdint.h,
- * if you want the limit (max/min) macros for int types.
+ * if you want the limit (max/min) macros for int types. 
```

Comment:
no need for trailing whitespace

Reply:
This file is automatically generated by the program flex. So we can ignore these whitespace issues.